public/app/features/browse-dashboards/components/DashboardsTree.tsx

```diff
@@ -73,24 +74,29 @@ export function DashboardsTree({
       Cell: CheckboxCell,
     };
 
+    // const nameColumnTranslation = t('dashboards-tree.browser.name-column','Name');
     const nameColumn: DashboardsTreeColumn = {
       id: 'name',
       width: 3,
-      Header: <span style={{ paddingLeft: 24 }}>Name</span>,
+      Header: (
+        <span style={{ paddingLeft: 24 }}>
+          <Trans i18nKey="dashboards-tree.browser.name-column">Name</Trans>
```

Comment:
And in this, just for consistency also start the keys with `browse-dashboards`. Don't think the `browser` in the middle is needed so we can remove it from there 👍